features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelperTest.kt

```diff
@@ -179,10 +182,296 @@ internal class DefaultImageWireframeHelperTest {
         )
     }
 
-    // region createImageWireframe
+    @Test
+    fun `M use intrinsic dimensions W createImageWireframeByBitmap { inset drawable }`(
```

Comment:
this test and the next one should be probably the part of the `createImageWireframeByDrawable` region which is declared at some part below